tests/integration-tests/tests/iam/test_iam.py

```diff
@@ -590,6 +590,7 @@ def _create_permission_boundary(permission_boundary_name):
                         "cloudwatch:DeleteDashboards",
                         "cloudwatch:GetDashboard",
                         "cloudwatch:PutMetricAlarm",
+                        "cloudwatch:PutCompositeAlarm",
```

Comment:
Let's check if the content is the same of https://github.com/aws/aws-parallelcluster/pull/5764 and if we can avoid to do the same kind of changes in multiple places.